### PR TITLE
Add Slack notifier, remove Campfire notifier

### PR DESCRIPTION
**Pull request by yaychris**
* Removed private URL
* Removed dependency on `active_support/core_ext`

### CODE REVIEW

File: lib/viget/deployment/slack_notifier.rb [hunk 1]
```diff
@@ -0,0 +1,154 @@
+require 'net/http'
+require 'uri'
+require 'json'
+
+module Viget
+  module Deployment
+    class SlackNotifier
+      attr_reader :cap, :slack_url, :slack_channel
+
+      def initialize(cap, slack_url:, slack_channel:)
```
Comment (efatsi) [inline lib/viget/deployment/slack_notifier.rb]: shnazzy

Comment (reagent) [inline lib/viget/deployment/slack_notifier.rb]: What does that do?

Comment (efatsi) [inline lib/viget/deployment/slack_notifier.rb]: required arguments - http://robots.thoughtbot.com/ruby-2-keyword-arguments

**Comment by mackermedia** [comment]
:+1: Nice work!

File: lib/viget/deployment/slack_notifier.rb (hunk 1 above)
Comment (zporter) [inline lib/viget/deployment/slack_notifier.rb]: I approve

**Comment by zporter** [comment]
This looks real good :+1:

**Comment by reagent** [comment]
Does this mean that I have to add the hook endpoint URL for every app that uses this deployment gem? If so, what do you think about just configuring the token part of it (kinda have no idea what I'm talking about)?

**Comment by yaychris** [comment]
Yes, you'll have to configure the URL yourself. We could go that route if people think it's best. I don't think it buys us much, though—most of the URL is the token part, and people will have to know to split it apart.

**Pull request merged**